### ISSUE TITLE
Game performance degrades as ones progresses in the game

### ISSUE THREAD

**Issue by Unrepentant-Atheist** [cataclysmbnteam/Cataclysm-BN#3531]
### Describe the bug

https://nowtransfer.de/4fa773c80efd . So I've got my game with 2 save here for y'all to performance analyze. `Randomville_slow` is my normal game. It's where the game slowed down. At some point in the game, the games slows down and there's no speeding it up anymore, more or less.

`Randomville_faster` is where the game is faster. I have deconstructed my DeathBus, spread out all my stuff evenly as to not tile overflow.

See more in Steps to Reproduce

### Steps To Reproduce

1. Play save `Randomville_slow` to get a feel how slow the game and have a comparison
2. Play `Randomville_faster`, Read a book or pass time and you'll notice time moves faster compared to `Randomville_slow`. (The green  "/" thing).
3. Move far away from my spread out time pile of stuff, middle of nowhere preferably where nothing is nearby, read a book or pass time, and you'll notice time moves even so faster
4. Now drop the 70 cash cards from inventory, read a book or pass time, and you'll notice time moves even faster. It's almost as fast as if it were a fresh game just created. (The green "/" thing when reading or passing time spins absurdely fast in a fresh, new, game)

### Versions and configuration

- OS: Windows
    - OS Version: 10.0.19045.3570 (22H2)
- Game Version: bc909ac [64-bit]
- Graphics Version: Tiles
- LAPI Version: 1
- Game Language: English [en_US]
- Mods loaded: [
    Bright Nights [bn],
    SpeedyDex [speedydex],
    Stats Through Kills [stats_through_kills],
    Stats Through Skills [StatsThroughSkills],
    Bionic Slots [cbm_slots],
    Elevated bridges [elevated_bridges],
    Cataclysm++ [Cata++],
    Aftershock [aftershock],
    PKs Rebalancing [PK_REBALANCE],
    Secronom [secronom],
    Secronom+ Lore Expansion [secronom_lore_expansion],
    C.R.I.T. Expansion [crt_expansion],
    Sierra Madre [sierra_madre],
    Bio-Synth Robot expansion [gekkobots],
    Nechronica Redux [Nechronica_Redux],
    Magiclysm [magiclysm],
    Really Dark Skies [realdarkskies],
    Arcana and Magic Items [Arcana],
    Project: Pack Rat [Jake's_Squirrel_mod],
    Add Bandits Extended+ [GOV_BANDITS_KAI_R],
    NPC Increased Location Spawns Combined [npc_increased_location_comb],
    More Bandits, Testing version [More_Bandits],
    More Dangerous Encounters [More_Dangerous_Encounters],
    Vampiric Stuff [vamp_stuff],
    Vampiric Stuff + Arcana Mod Extension [vamp_stuff+arcana],
    Magiclysm Revamp and Additions [magicrevamp],
    Miscellaneous Magiclysm Expansions [demon_spider_a],
    Urban Development [Urban_Development],
    Morlock Village [morlock_village],
    Arcana/Cata++ Patchmod [Arcana_Cata++_Patch],
    Arcana/Aftershock Patchmod [Arcana_aftershock_Patch],
    Arcana/C.R.I.T. Patchmod [Arcana_Crit_Patch],
    Arcana/Magiclysm Patchmod [Arcana_Magiclysm_Patch],
    Really Dark Skies plus Arcana [realdarkskies_Arcana_patchmod],
    Really Dark Skies plus Cataclysm++ [realdarkskies_CATA_patchmod],
    Really Dark Skies plus Aftershock [realdarkskies_aftershock_patchmod],
    Really Dark Skies plus PKs Rebalancing [realdarkskies_PK_REBALANCE_patchmod],
    Really Dark Skies plus Secronom [realdarkskies_secronom_patchmod],
    Really Dark Skies plus C.R.I.T. [realdarkskies_CRIT_patchmod],
    Really Dark Skies plus Bio-Synth Robot Expansion [realdarkskies_gekkobots_patchmod],
    Really Dark Skies plus magiclysm [realdarkskies_magiclysm_patchmod],
    Really Dark Skies plus Morlock Village [realdarkskies_morlock_village_patchmod],
    Even More Mutations [even_more_mutations],
    Mutant NPCs [mutant_npcs],
    Psychiclysm [psychiclysm],
    Modular Turrets with No Blacklist [modular_turrets_no_blacklist],
    Salvaged Robots [Salvaged_Robots],
    Bug's life [Bug's life],
    Mutant Animals [mutant_animals],
    Mutant Insects [mutant_insects],
    DinoMod [DinoMod],
    Arcana/DinoMod Patchmod [Arcana_Dinomod_Patch],
    Really Dark Skies plus DinoMod [realdarkskies_Dino_patchmod],
    Mythical Martial Arts [MMA],
    Mythical Martial Arts - Mods Extension [MMA_Mods_Extension],
    Wield Wild Weapons [WIELD_WILD_WEAPONS],
    Tankmod: Revived [Tankmod_Revived],
    Valik's Machine Shop 1.1 [valikengines],
    Thog's Vehicles [Thogs_Vehicles],
    Really Dark Skies plus Thog's Vehicles [realdarkskies_Thogs_Vehicles_patchmod],
    Thog's Things [Thogs_Things],
    CDDA Variety Pack [ks_variety_pack],
    Hydroponics [hydroponics],
    Wandering Masters [wandering_masters],
    Expanded Archery [ExArch],
    MST Extra [MST_Extra],
    Makeshift Items Mod [makeshift],
    Bionic Space Expander [Bionic_Space_Expander],
    Bionics Systems Recipes [bio_recipes],
    Caches [P_Caches],
    Dorf Life [Dorf_Life],
    Randomized Mall [randomized_mall],
    Tekitou Map Commercial Overhaul [tekitoumap],
    Beta National Guard Camp [national_guard_camp],
    Nuclear Power Plant [NPP],
    Mug NPCs [mug_NPCs],
    NPC Recruitment Options [recruitment_options],
    Necropolis Actually Findable [necro_find],
    Chorus's Arcana Addon [Chorus Arcana Addon],
    Chorus's Gear [Chorus_Weapons],
    Chorus's CBMs [Chorus CBMs],
    Chorus's Martial Arts [Chorus_MA],
    Chorus's Primitive Living [Chorus Primitive],
    Chorus's Starts [Chorus_Starts],
    Idzibear's Additions Testing Ground [Idzibear_Additions_test]
]

**Comment by Krwak** [comment]
You can upload saves directly in the bug report.

I didn't check the saves but try going away from your base 62 tiles and then compare performance. All items in base/mobile base slow down performance.

**Comment by Unrepentant-Atheist** [comment]
Yeah, probably is my hoarding problem why the game always slows down... guess it's not really a "bug" then :(.

**Comment by chaosvolt** [comment]
Some of that can be improved if so (like when https://github.com/cataclysmbnteam/Cataclysm-BN/pull/3489 is merged), but still test if your performance picks up when far away from your character's current stomping grounds.
